courses/machine_learning/deepdive2/structured/solutions/2_prepare_babyweight.ipynb

```diff
@@ -0,0 +1,829 @@
+{
```

Comment:
<p>Project listed twice</p><p>Also, os.environ["Project"] is redundant with the bash export above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/666/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Done</p>